python_modules/dagster/dagster/seven/__init__.py

```diff
@@ -14,3 +14,43 @@
     from io import StringIO  # pylint:disable=import-error
 else:
     from StringIO import StringIO  # pylint:disable=import-error
+
+
+# TODO implement a generic import by name -- see https://stackoverflow.com/questions/301134/how-to-import-a-module-given-its-name
+
+# https://stackoverflow.com/a/67692/324449
+def import_module_from_path(module_name, path_to_file):
```

Comment:
you also added this in airline_demo/utils.py - maybe meant to remove the one above?